packages/react/presence/src/Presence.tsx

```diff
@@ -52,7 +51,7 @@ function usePresence(present: boolean) {
         },
       },
       unmountSuspended: {
-        on: { ANIMATION_END: 'unmounted', TRANSITION_END: 'unmounted' },
+        on: { MOUNT: 'mounted', ANIMATION_END: 'unmounted', TRANSITION_END: 'unmounted' },
```

Comment:
> - Potential bug fix I spotted which helped clean up some logic: Adds `MOUNT: 'mounted'` event to `unmountSuspended` state since users should be able to re-mount during the animation/transition phase.

Yep 🙂